tests/unit/utils/css-transitions/on-transition-end-test.js

```diff
@@ -0,0 +1,121 @@
+/* eslint no-magic-numbers:0 */
+import { module, test } from 'qunit';
+import onTransitionEnd from 'dummy/utils/css-transitions/on-transition-end';
+import sinon from 'sinon';
+import { createElement, clearScenario, setAnimationStyle } from './helpers';
+
+const { spy } = sinon;
+
+module('Unit | Util | css-transitions | on-transition-end', {
+	afterEach() {
+		clearScenario();
+	}
+});
+
+test('it waits for any element transition', (assert) => {
+	const done = assert.async();
+	const element = createElement();
+	const fn = spy();
+
+	element.style.transition = 'all .5s linear 0s';
+
+	onTransitionEnd(element, fn);
+
+	setAnimationStyle(element, 'opacity', '0.5');
+
+	setTimeout(() => {
+		assert.ok(fn.notCalled, 'fn is not called');
+	}, 250);
+
+	setTimeout(() => {
+		assert.ok(fn.calledOnce, 'fn is called once');
+		done();
+	}, 600);
+});
+
+test('it calls function with transition event', (assert) => {
+	const done = assert.async();
+	const element = createElement();
+	const fn = spy();
+
+	element.style.transition = 'all .25s linear 0s';
+
+	onTransitionEnd(element, fn);
+
+	setAnimationStyle(element, 'opacity', '0.5');
+
+	setTimeout(() => {
+		assert.ok(fn.calledOnce, 'fn is called once');
+		assert.ok(fn.args[0][0] instanceof Event, 'fn is called with event');
+		done();
+	}, 300);
+});
+
+test('it waits for selected element transition', (assert) => {
```

Comment:
What purpose? I don't need to test css-transitions. I just want to work with them.